espnet/nets/pytorch_backend/transformer/encoder.py

```diff
@@ -25,7 +25,10 @@
     PositionwiseFeedForward,  # noqa: H301
```

Comment:
FYI, with the conformer implementation, we have several variations for this subsampling.
We'll do some refactoring soon.